### DISCUSSION TITLE
How to change units in memory widget?

### DISCUSSION THREAD

**Discussion by VD171** [ClementTsang/bottom#695] [Q&A] [answered]
Well, first, THANK YOU VERY MUCH for this amazing app.
I love it ❤️

How can I change the units in memory widget?
It's showing GiB, but I want to see MiB.
Thank you very much

**Comment by ClementTsang** [comment, marked as answer]
Currently, there's no manual way of changing the prefix type (it's automatically set based on total memory available), but I can look into adding options for that when I have some time.

Issue: https://github.com/ClementTsang/bottom/issues/696

**Reply by VD171** [reply]
Great ! I would love for your care. Your app is wonderful, I was using htop. I tried gotop too, but bottom is the best of all. Thank you very much again ❤️
